tool/tsh/device.go

```diff
@@ -0,0 +1,91 @@
+// Copyright 2022 Gravitational, Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"fmt"
+
+	"github.com/gravitational/kingpin"
+	"github.com/gravitational/trace"
+
+	devicepb "github.com/gravitational/teleport/api/gen/proto/go/teleport/devicetrust/v1"
+	"github.com/gravitational/teleport/lib/client"
+	"github.com/gravitational/teleport/lib/devicetrust"
+	"github.com/gravitational/teleport/lib/devicetrust/enroll"
+)
+
+type deviceCommand struct {
+	enroll *deviceEnrollCommand
+}
+
+func newDeviceCommand(app *kingpin.Application) *deviceCommand {
+	root := &deviceCommand{
+		enroll: &deviceEnrollCommand{},
+	}
+
+	// "tsh device" command.
+	parentCmd := app.Command(
+		"device", "Manage your device. Requires Teleport Enterprise.")
```

Comment:
Nit: maybe "this" instead of "your" makes it clearer what this command does especially in the `enroll` help.

Reply:
Thanks, done.